sympy/solvers/bivariate.py

```diff
@@ -210,7 +210,7 @@ def _solve_lambert(f, symbol, gens):
         rhs = log(rhs)
 
     lhs = factor(lhs, deep=True)
-    # make sure we are inverted as completely as possible
+    # make sure we have inverted as completely as possible
```

Comment:
It will be better if this line can be further changed, highlighting what is being inverted here. Just by reading this, it looks confusing.